src/js/common/schemaform/ObjectField.jsx

```diff
@@ -47,6 +49,17 @@ class ObjectField extends React.Component {
     this.onPropertyChange = this.onPropertyChange.bind(this);
     this.onPropertyBlur = this.onPropertyBlur.bind(this);
     this.isRequired = this.isRequired.bind(this);
+    // This runs a series of steps that order properties and then group them into
+    // expandable groups. If there are no expandable groups, then the end result of this
+    // will be an array of single item arrays
+    this.orderAndFilterProperties = _.flow(
```

Comment:
`_.flow` creates a function to process the list of object properties and it doesn't need to be recreated every time we render or props change, just run again. So this seemed more efficient. 

This is the type of code I would normally write as a class method, but that wouldn't work well with `_.flow`, which I thought made the code clearer.